pkg/driver/driver_azure.go

```diff
@@ -713,6 +713,10 @@ func (clients *azureDriverClients) getRelevantVMs(ctx context.Context, machineID
 
 	if len(machines) > 0 {
 		for _, server := range machines {
+			if !verifyAzureTags(server.Tags, searchClusterName, searchNodeRole) {
+				continue
```

Comment:
Can we add the logs here saying: 
"XYZ nic/disk/vm was found but not verified with tags.." ? 
- This will help us understand if we are missing out on any resources.

Reply:
`V(4)`?

Reply:
Ah V(2) will be better, we would like to observe, if we see any issues as in past, we can demote it later.

Reply:
Done.